etlman/projects/urls.py

```diff
@@ -1,9 +1,12 @@
 from django.urls import path
 
-from etlman.projects.views import step_form_upsert_view
+from etlman.projects.views import new_project_wizard_view, step_form_upsert_view
 
 app_name = "projects"
 urlpatterns = [
     path("step_form/", view=step_form_upsert_view, name="step_form_upsert"),
     path("step_form/<str:pk>/", view=step_form_upsert_view, name="step_form_upsert"),
+    path(
+        "add_project_wizard/", view=new_project_wizard_view, name="add_project_wizard"
```

Comment:
Nitpick:
```suggestion
        "new-project/", view=new_project_view, name="new_project"
```
Reasoning:
- This view is not really part of a wizard (it's a one-off task)
- Dashes are generally preferred to underscores in URLs (mostly [for SEO](https://developers.google.com/search/docs/advanced/guidelines/url-structure#:~:text=Consider%20using%20hyphens%20in%20your,concepts%20in%20the%20URL%20easier.&text=We%20recommend%20that%20you%20use,(%20_%20)%20in%20your%20URLs.) which is not an issue for us, so this is purely for aesthetics / convention)